src/log.js

```diff
@@ -219,6 +219,7 @@ export class Log {
    * @param {...*} var_args Arguments substituted into %s in the message.
    * @return {T} The value of shouldBeTrueish.
    * @template T
+   * @throws {Error}
```

Comment:
should i remove here then? didnt realize the compiler doesnt even use it besides detecting side effects